content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -254,3 +254,8 @@ Example: `node.cloudprovider.kubernetes.io/shutdown:NoSchedule`
 
 If a Node is in a cloud provider specified shutdown state, the Node gets tainted accordingly with `node.cloudprovider.kubernetes.io/shutdown` and the taint effect of `NoSchedule`.
 
+## kubectl.kubernetes.io/default-container
+
+Example: `kubectl.kubernetes.io/default-container:container1`
+
+If a Pod has the annotation `kubectl.kubernetes.io/default-container`, consumers (CLI, operators) know which is the default container. For example, `kubectl logs` or `kubectl exec` without `-c` or `--container` flag will use this default container. 
```

Comment:
1. We're listing this in the wrong place (under a heading “**The taints listed below are always used on Nodes**”)
1. We should explicitly say that the value of the annotation is the container _name_ that is default for this Pod.